.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,51 @@
+name: "CodeQL"
+
+on:
+  schedule:
+    - cron: '40 21 * * 2'
```

Comment:
I forget how to read these expressions -- what'd be the frequence of timed runs?

Reply:
As per https://crontab.cronhub.io/ apparently "At 09:40 PM, only on Tuesday", i.e. once per week. Which sounds good.